net/src/p2p_config.rs

```diff
@@ -142,7 +170,45 @@ impl P2pConfig {
     }
 }
 
-// statics
+/// end_user config
+impl P2pConfig {
+    pub fn default_end_user_config() -> serde_json::Value {
+        json!({
+          "webproxy": {
+            "connection": {
+              "rsaBits": 1024,
+              "bind": [
+                "wss://0.0.0.0:0/"
+              ]
+            },
+            "wssAdvertise": "auto",
+            "wssRelayPeers": null
+          }
+        })
+    }
+
+    pub fn load_end_user_config(
+        maybe_end_user_config_filepath: Option<String>,
+    ) -> serde_json::Value {
```

Comment:
Should this just return a `String` so we don't have to do the json parsing / have the `json!` macro in the default, etc?

Reply:
I believe `serde_json::Value` is better as we want to make sure as early as possible that the config is valid JSON.